scripts/sct_image.py

```diff
@@ -276,12 +276,12 @@ def main(args=None):
                 fname_out.append(add_suffix(fname_in[0], '_' + dim_list[dim].upper() + str(i).zfill(4)))
```

Comment:
This is not new code, but constructing such strings could be done way more easily with `str.format` or `str.__mod__`!